src/compiler/symb.bas

```diff
@@ -1977,6 +1987,124 @@ function symbCheckConstAssign _
 	function = TRUE
 end function
 
+function symbCheckConstAssignFuncPtr _
```

Comment:
This function seems very similar to `symbCalcProcMatch()`, it would be nice to share some of the code. (I'm thinking of a generic function signature comparison/compatibility check)

Reply:
yea, that's been on my mind as well.  Maybe you can see from the changes I've made to the match score, I'm trying to head in that direction.  The idea is to give additional meaning to the match score.  Possibly even embedding information in the least significant bits.